### PR TITLE
Whatsnew 80 sting additions

### PR DESCRIPTION
**Pull request by Errolyn**
Strings for whatsnew-80

they are not up on dev yet but should be soon.

demo link: https://www-demo-pr-9291.herokuapp.com/en-US/firefox/80.0/whatsnew/all/

pr: https://github.com/mozilla/bedrock/pull/9291

### CODE REVIEW

**Comment by Errolyn** [comment]
@peiying2 I think this is ready to look at :D

File: en/firefox/whatsnew/whatsnew-fx80.ftl [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+# content buckets below hero
+whatsnew80-password-fatigue = Password fatigue is real
+whatsnew80-the-average-person = The average person has 130 online accounts. { -brand-name-firefox } for mobile remembers and stores your passwords safely across synced devices, so you can log in with just one click.
+
+whatsnew80-privacy-you-never = Privacy you never have to ponder
+whatsnew80-firefox-for-mobile = { -brand-name-firefox } for mobile blocks most online ad trackers automatically so there’s no need to dig into your security settings.
+
+whatsnew80-new-firefox-android = New { -brand-name-firefox } for Android
+whatsnew80-our-latest-version = Our latest version of { -brand-name-firefox } features lighting-fast page loads and a clean new design that makes it easier to get more things done, more quickly.
```
Comment (peiying2) [inline en/firefox/whatsnew/whatsnew-fx80.ftl:25]: `lighting-fast` should be `lightening-fast`.

File: en/firefox/whatsnew/whatsnew-fx80.ftl [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+# html page title
+whatsnew80-page-title = What’s new with { -brand-name-firefox }
+
+whatsnew80-up-to-date-banner = Congrats! You’re using the latest version of { -brand-name-firefox }.
+
+# Main title
+whatsnew80-main-title = Do just about everything, a little faster
+whatsnew80-main-intro = Move from computer to phone without missing a beat. Get { -brand-name-firefox } for mobile.
+whatsnew80-primary-cta-description = Enter your email or phone number for a download link
```
Comment (peiying2) [inline en/firefox/whatsnew/whatsnew-fx80.ftl:15]: This string doesn't match what's on the demo: `Enter your email or phone number`

File: en/firefox/whatsnew/whatsnew-fx80.ftl (hunk 1 above)
Comment (Errolyn) [reply en/firefox/whatsnew/whatsnew-fx80.ftl:25]: Good catch!

File: en/firefox/whatsnew/whatsnew-fx80.ftl (hunk 2 above)
Comment (Errolyn) [reply en/firefox/whatsnew/whatsnew-fx80.ftl:15]: It's still using the default string, I need to push up what we are overriding it with should be the same here in a moment.

File: en/firefox/whatsnew/whatsnew-fx80.ftl [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+# content buckets below hero
+whatsnew80-password-fatigue = Password fatigue is real
+whatsnew80-the-average-person = The average person has 130 online accounts. { -brand-name-firefox } for mobile remembers and stores your passwords safely across synced devices, so you can log in with just one click.
+
+whatsnew80-privacy-you-never = Privacy you never have to ponder
+whatsnew80-firefox-for-mobile = { -brand-name-firefox } for mobile blocks most online ad trackers automatically so there’s no need to dig into your security settings.
+
+whatsnew80-new-firefox-android = New { -brand-name-firefox } for Android
```
Comment (peiying2) [inline en/firefox/whatsnew/whatsnew-fx80.ftl:24]: Missed this one. `Android` is a brand

**Pull request merged**